bin/terminus

```diff
@@ -12,22 +12,16 @@
 // Unset memory limit
 ini_set('memory_limit', -1);
 
-// Version compatibility check and messaging
 if (version_compare(PHP_VERSION, '7.4.0', '<') === true) {
     echo "\n";
     echo 'Sorry, your PHP version (' . PHP_VERSION . ') is no longer supported.' . "\n";
     echo 'Upgrade to PHP 7.4 or newer to use Terminus 3. For PHP versions prior to 7.4, downgrade to Terminus 2.x.'; echo "\n\n";
     exit(1);
-} elseif (version_compare(PHP_VERSION, '8.2.0', '>=') === true) {
```

Comment:
We should always have some version of PHP that we don't support yet. Is it too early to say that 8.2 stable will run without issue on the next release of Terminus? If not, then let's at a minimum reject 8.3.0+

Reply:
@greg-1-anderson Rolled-back to support active php 8.1 only